inlong-sort/sort-core/src/main/java/org/apache/inlong/sort/flink/hive/HiveMultiTenantWriter.java

```diff
@@ -126,6 +147,20 @@ public void processElement(SerializedRecord serializedRecord, Context context,
 
             hiveWriter.processElement(recordTransformer.toRecord(serializedRecord).getRow(),
                     proxyContext.setContext(context), collector);
+
+            if (auditImp != null) {
+                Pair<String, String> groupIdAndStreamId = inLongGroupIdAndStreamIdMap.getOrDefault(
+                        serializedRecord.getDataFlowId(),
+                        Pair.of("", ""));
+
+                auditImp.add(
+                        Constants.METRIC_AUDIT_ID_FOR_OUTPUT,
+                        groupIdAndStreamId.getLeft(),
+                        groupIdAndStreamId.getRight(),
+                        serializedRecord.getTimestampMillis(),
```

Comment:
ditto

Reply:
sinkRecord.getTimestampMillis() is event time extracted from pulsar/tube message